spec/factories/legal_aid_applications.rb

```diff
@@ -627,6 +627,13 @@
       end
     end
 
+    trait :with_cfe_v3_result do
```

Comment:
There are a few places where tests use the `with_cfe_v2_result` trait. I wonder if it's worth duplicating those tests and making sure they pass using the new `with_cfe_v3_result` trait too.

For example `spec/requests/providers/means_reports_spec.rb`, `spec/services/reports/means_report_creator_spec.rb` and several CCMS specs. 

Maybe it's overkill though.

Reply:
Yeah I noticed those too, but tbh I was thinking we should leave those and replace the traits for those when we make V3 live. But maybe you are right, in case there are any breakages maybe worth sorting now and then we know we won't be blocked by it when live.